media-gfx/gscan2pdf/gscan2pdf-1.7.1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DIST_TEST="do"
+
+inherit eutils perl-module
+
+DESCRIPTION="Scan documents, perform OCR, produce PDFs and DjVus"
+HOMEPAGE="http://gscan2pdf.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="test"
+
+DEPEND="test? (
+		app-text/pdftk
+		app-text/poppler
+		dev-perl/Sub-Override
+		dev-perl/Test-Pod
+		media-libs/fontconfig
+		x11-libs/gdk-pixbuf[tiff]
+
+		app-text/djvu[tiff]
+		app-text/tesseract[-opencl,osd,tiff]
+		app-text/unpaper
+		media-gfx/imagemagick[-opencl,djvu,png,tiff,perl]
+		media-gfx/sane-backends[sane_backends_test]
+		media-gfx/sane-frontends
+	)"
+# Tests fail on sandbox violation with OpenCL ImageMagick and Tesseract
```

Comment:
I don't think it's appropriate to force `-opencl` flag on `tesseract` and `imagemagick` just because the tests fail otherwise. Can you elaborate more on the sandbox violation problem? Can we selectively disable those tests, or just restrict the tests altogether instead? Forcing `-opencl` has a larger impact on the user's system.

Minor nitpick: the indentation level should be 1 for DEPEND items, not 2.